### PR TITLE
Fixes to upstream to work in our environment (OPS-6044)

### PR DESCRIPTION
**Pull request by GabeL7r**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -153,7 +153,7 @@ rhel7cis_rule_2_5: true
 # Section 3 is Network Configuration (Disable unused network protocols, Network parameters (host), Network parameters (Host and Router), Uncommon Network Protocols, Firewall Configuration, and Configure iptables)
 rhel7cis_rule_3_1_1: true
 rhel7cis_rule_3_1_2: true
-rhel7cis_rule_3_2_1: true
+rhel7cis_rule_3_2_1: false
```
Comment (GabeL7r) [inline defaults/main.yml:156]: This is IP Forwarding disabled rule... we want to keep it enabled

File: local.yml [hunk 2]
```diff
@@ -7,5 +7,5 @@
       is_container: false
   roles:
       - role: "{{ playbook_dir }}"
-        rhel7stig_system_is_container: "{{ is_container | default(false) }}"
-        rhel7stig_ssh_required: "{{ is_container == false  }}"
+        rhel7cis_system_is_container: "{{ is_container | default(false) }}"
```
Comment (GabeL7r) [inline local.yml:10]: Seemed like copy pasta fail on the upstream part

File: tasks/main.yml [hunk 3]
```diff
@@ -19,7 +22,7 @@
 - name: Check rhel7cis_bootloader_password_hash variable has been changed
   assert:
     that: rhel7cis_bootloader_password_hash != 'grub.pbkdf2.sha512.changethispassword'
-    msg: "This role will not be able to run single user password commands as rhel7stig_bootloader_password_hash variable has not been set"
+    msg: "This role will not be able to run single user password commands as rhel7cis_bootloader_password_hash variable has not been set"
```
Comment (GabeL7r) [inline tasks/main.yml:25]: Copy pasta fail

File: tasks/prelim.yml [hunk 4]
```diff
@@ -55,7 +55,7 @@
 
 - name: "PRELIM | 6.2.8 | Set fact for home directory paths for interactive users"
   set_fact:
-    rhel_07_stig_interactive_homedir_results: "{{ rhel_07_6_2_8_getent.stdout_lines }}"
+    rhel_07_cis_interactive_homedir_results: "{{ rhel_07_6_2_8_getent.stdout_lines }}"
```
Comment (GabeL7r) [inline tasks/prelim.yml:58]: Copy pasta fail

File: tasks/section_6/cis_6.1.x.yml [hunk 5]
```diff
@@ -212,8 +212,7 @@
   - name: "SCORED | 6.1.12 | AUDIT | Ensure no ungrouped files or directories exist | Displaying all ungrouped files or directories"
     debug:
       msg: "Manual intervention is required -- missing group on items in {{ item.item.mount }}: {{ item.stdout_lines | join(', ') }}"
-    changed_when:
-    - rhel7stig_audit_complex
+        #changed_when: - rhel7stig_audit_complex
```
Comment (GabeL7r) [inline tasks/section_6/cis_6.1.x.yml:215]: I couldn't find `rhel7stig_audit_complex` anywhere so I just commented this out.

File: tasks/section_6/cis_6.2.x.yml [hunk 6]
```diff
@@ -124,7 +124,7 @@
       debug: 
         msg: "You will need to mkdir -p {{ item }} and chown properly to the correct owner and group."
       with_items: "{{ rhel_07_6_2_5_audit | map(attribute='dir') | list }}"
-      changed_when: rhel7cis_audit_complex
+        #changed_when: rhel7cis_audit_complex
```
Comment (GabeL7r) [inline tasks/section_6/cis_6.2.x.yml:127]: Same in that I couldn't find `rhel7cis_audit_complex` anywhere

**Review by cheethoe — APPROVED** [review]

**Pull request merged**